### DISCUSSION TITLE
Invoice export for sage

### DISCUSSION THREAD

**Discussion by Gazzxy** [YetiForceCompany/YetiForceCRM#14858] [How to]
is there anyway to customise how the system exports CSV's ? looking to just export some accounting data I want ready to go right into sage.. and have it setup so your accountant can do it without having to adjust the csv first

**Comment by MexasPT** [comment]
Yes... send me a PM.

**Reply by Gazzxy** [reply]
ok sorry busy week.. am not seeing a way to send a pm

**Reply by Gazzxy** [reply]
yeh there isnt a way to send a pm here.. anyway to just point me in the right direction here

**Comment by Gazzxy** [comment]
?
